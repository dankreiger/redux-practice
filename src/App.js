import React, { Component } from 'react';

import Flashcard from './containers/Flashcard/Flashcard';

class App extends Component {
  render() {
    return (
      <div className="App">
        <Flashcard />
      </div>
    );
  }
}

export default App;
