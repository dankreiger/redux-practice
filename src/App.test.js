import React from 'react';
import App from './App';

describe(App, () => {
  let app;
  beforeEach(() => {
    app = shallow(<App />);
  });

  it('renders correctly', () => {
    expect(toJson(app)).toMatchSnapshot();
  });
});
