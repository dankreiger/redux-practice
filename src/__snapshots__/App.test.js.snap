// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`class App extends _react.Component {
  render() {
    return (
      _react2.default.createElement('div', { className: 'App', __source: { fileName: _jsxFileName, lineNumber: 8 }, __self: this },
        _react2.default.createElement(_Flashcard2.default, { __source: { fileName: _jsxFileName, lineNumber: 9 }, __self: this })));


  }} renders correctly 1`] = `
<div
  className="App"
>
  <Connect(Flashcard) />
</div>
`;
